.github/workflows/ci-unit-broker-broker-gp2.yaml

```diff
@@ -58,8 +58,11 @@ jobs:
               - '*.md'
               - '**/*.md'
 
+      - name: print env
+        run: echo ${{ steps.changes.outputs.all_count  }} && echo ${{ steps.changes.outputs.docs_count  }} && echo ${{ steps.changes.outputs.docs  }} && echo ${{steps.changes.outputs.all_count > steps.changes.outputs.docs_count}}
```

Comment:
This echo shows some of the values that are interpolated by GitHub. View them in the action's logs.